packages/cosmic-swingset/src/chain-main.js

```diff
@@ -319,12 +324,15 @@ export default async function main(progname, args, { env, homedir, agcc }) {
     function doOutboundBridge(dstID, msg) {
       const portNum = portNums[dstID];
       if (portNum === undefined) {
+        const [portKey] = Object.entries(BRIDGE_KEYS).find(
+          ([_, d]) => d === dstID,
+        ) || [`${dstID}${PORT_SUFFIX}`];
```

Comment:
minor readability suggestion
```suggestion
        const portKey = Object.keys(BRIDGE_KEYS).find(
          k => BRIDGE_KEYS[key] === dstID,
        ) || [`${dstID}${PORT_SUFFIX}`];
```

modulo other requests